### DISCUSSION TITLE
[LIMITATION] Issues with Deploying MTAs on Apple Sillicon

### DISCUSSION THREAD

**Discussion by lechnerc77** [SAP-archive/btp-setup-automator#491] [General]
We recently came across issues when deploying MTA apps via the BTP Setup Automator when running on Apple Silicon (M1/M2).
The setup of the resources on BTP works as expected. However, once you want to deploy an MTA archive via the CF CLI plugin you will run into an error like: 

```bash
fatal: morestack on g0
SIGTRAP: trace trap
PC=0x80adb62 m=1 sigcode=128

goroutine 0 [idle]:
runtime.abort()
        /usr/local/go/src/runtime/asm_386.s:821 +0x2 fp=0x8c40618 sp=0x8c40614 pc=0x80adb62
runtime.morestack()
        /usr/local/go/src/runtime/asm_386.s:416 +0x24 fp=0x8c4061c sp=0x8c40618 pc=0x80ac9a4

goroutine 1 [running, locked to thread]:
        goroutine running on other thread; stack unavailable
```

Building the BTP Setup Automator manually does not resolve the issue either as other errors occur when building the Docker image and installing the CF plugin for MTA.

We will use this discussion once we have new information on that topic.

**Comment by ublock** [comment]
Resolved by https://github.com/SAP-samples/btp-setup-automator/pull/495
